### ISSUE TITLE
slow bootstrapping on travis containers

### ISSUE THREAD

**Issue by andreineculau** [tobiipro/support-firecloud#53]
after #51 , the first run on a travis containers (with no sudo) takes 30+ minutes. Thanks to aggressive caching, the following runs go down 10 times, to 3+ minutes.

I wrote TravisCI a feature request that should substantially decrease the first-time runs, and also the cache sizes.

---

<blockquote>
Hi!

We love the quick-to-spin no-sudo containers, yet there's a BUT :)
We run on macos when developing, but we deploy to linux. And homebrew is king on macos for package management.
Rather than finding the equivalent apt packages, or configure-make-etc ourselves, or finding out quirks with platform-dependent python2 pip2 python3 pip3 etc, we have recently switched our bootstrapping to http://linuxbrew.sh

You can actually see how on the first run (no cache) it is 4 times faster to spin up a sudo-VM and install more packages
than it is to spin up a no-sudo-containers and install less packages via linuxbrew
https://travis-ci.com/tobiipro/support-firecloud/builds/93344821

It's only thanks to aggressive caching that we manage to go down to 3-4 minutes on following runs on no-sudo-containers...

The background of this behaviour is that
just like homebrew builds bottles assuming that HOMEBREW_PREFIX is /usr/local,
linuxbrew assumes HOMEBREW_PREFIX is /home/linuxbrew/.linuxbrew

On no-sudo travis containers /home/linuxbrew is not writable, so linuxbrew then defaults to ~/.linuxbrew i.e. /home/travis/.linuxbrew but many bottles are prefix-dependent so linuxbrew cannot use those bottles, and instead compiles locally. Which can take forever even for simple packages like git, bash and make.

So here's a feature request - allow /home/linuxbrew/to be writable on travis containers
as per the linuxbrew's install script https://github.com/Linuxbrew/install/blob/master/install#L236
and then all bottles would be poured in a zippy.

Thank you in advance,
Andrei
</blockquote>

ref: ticket number 1340

**Comment by andreineculau** [comment]
a reply from travis referenced a recent blog post https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

TL;DR remove `sudo-false` because containers will go away (in 3 days from today actually)

PS: no idea who received this "service mail" they talk about though . do you @alsmyr ?

> 28 November, 2018 - We will send a service email to remind folks still using sudo: false on recent builds to remind you to migrate.

**Comment by alsmyr** [comment]
Not seen any "service mail". Our billing email have nothing of that either.
